examples/tutorials/tutorial-1-vector.ipynb

```diff
@@ -200,6 +200,31 @@
     "norm_max = x.norm(normtype=libceed.NORM_MAX)\n",
     "print(norm_max)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* If you have installed libCEED with CUDA support and Numba, you can use device memory in your `CeedVector`s. In the following example, we create a `CeedVector` with a libCEED contex that supports CUDA, associate the data stored in a CeedVector with a `numpy.array`, and get a Numba `DeviceNDArray` containing the data on the device."
```

Comment:
I see there was prior use of this type, but `CeedVector` doesn't have specific meaning to a Python user, which may think of `Ceed.Vector` or `libceed.Vector`.

Reply:
Changed across all notebooks.